plugins/org.python.pydev.core/src/org/python/pydev/core/ICompletionState.java

```diff
@@ -119,6 +119,8 @@
 
     ICompletionState getCopyWithActTok(String value);
 
+    ICompletionState getCopyWithActTok(String value, int line, int col);
```

Comment:
Given that sometimes the line/col is 0 or 1-based, there should be a docstring saying whether this index is 0 or 1 based.

-- Note: the getters say it's 0-based, so, I expect those to be 0-based here.

     * @return the line for the request (starting at 0)
     */
    int getLine();

    /**
     * @return the col for the request (starting at 0)
     */
    int getCol();